perl-alien/lib/Alien/YAMLScript.pm

```diff
@@ -7,4 +7,10 @@ use warnings;
 
 use parent 'Alien::Base';
 
+our $libyamlscript_version = '0.1.34';
+
+die "YAMLScript::Alien $VERSION requires libyamlscript $libyamlscript_version" .
```

Comment:
The distribution name is wrong
```suggestion
die "Alien::YAMLScript $VERSION requires libyamlscript $libyamlscript_version" .
```

Reply:
Fixed the dist name thing.

The error message says: `"but you have " . __PACKAGE__->version` which I think is correct.